topi/python/topi/arm_cpu/conv2d.py

```diff
@@ -280,13 +280,15 @@ def _schedule_spatial_pack(cfg, s, data_vec, kernel_vec,
     s[conv].compute_at(s[last], ow)
 
     # mark parallel
-    s[last].parallel(co)
+    p = s[last].fuse(n, co)
+    s[last].parallel(p)
 
     if data_vec.op.name == 'data_vec_undilated':
-        _, h, _, _, _, _, _, _ = s[data_vec].op.axis
+        n, h, _, _, _, _, _, _ = s[data_vec].op.axis
     else:
-        _, h, _, _, _, _ = s[data_vec].op.axis
-    s[data_vec].parallel(h)
+        n, h, _, _, _, _ = s[data_vec].op.axis
```

Comment:
Why not just use `n = s[data_vec].op.axis[0]` and `h=s[data_vec].op.axis[1]` and remove the if/else?

Reply:
I actually prefer the way it is written now. It is more clear for us to understand the ndim difference.